### ISSUE TITLE
Create official postgres-operator-cluster helm chart

### ISSUE THREAD

**Issue by Whazor** [zalando/postgres-operator#1846]
A `postgres-operator-cluster` helm chart would be useful for creating new Postgres instances via helm and via this operator.

This helm chart already [exists](https://artifacthub.io/packages/helm/k8s-at-home/zalando-postgres-cluster) and is created by the k8s at home community. However, an official chart would be better.

The helm chart could contain extra features like creating teams if wanted by the user.

**Comment by ibotty** [comment]
See https://github.com/zalando/postgres-operator/blob/master/docs/quickstart.md#helm-chart
and #1812.

**Comment by Whazor** [comment]
I am proposing a new helm chart besides the existing two, with its soul purpose of creating new clusters. Build around the [Cluster manifest](https://postgres-operator.readthedocs.io/en/latest/reference/cluster_manifest/).

**Comment by ibotty** [comment]
Yeah, I misunderstood.  I don't quiet see why that might be useful though, but I am not in charge anyway.

**Comment by FxKu** [comment]
Maybe I misunderstanding it as well :smiley: The helm chart is there to create the postgres operator deployment. To create Postgres clusters you then submit YAML manifests. Why do it with helm? What's the benefit?

**Comment by Whazor** [comment]
The reasons I can think of:
- Helm charts can have dependencies, you can make creating instances dependent on rolling out the operator. In this case you would not get an error about missing CRD's.
- Certain places have standards with tooling that make deploying software build around Helm, for example [helmfile](https://github.com/helmfile/helmfile). In companies you might only have the permission to install helm charts as well.
- I have seen a helm chart for this in my company, but as stated in first post also someone else already created a helm chart already. I am curious if more people are interested in this, or that it might be too niche 😄

**Comment by jcsf** [comment]
I agree with @Whazor, having a helm chart to create the CRs is very useful and allows to create dependencies between the helm charts avoiding errors about missing CRD's. For example, in my company we are using Flux, therefore it would be very helpful to have this chart to use in Flux since it will ensure that the operator is installed before installing the CRs. You can see this approach already being followed by several other operators, just two examples:
- Minio Operator: https://github.com/minio/operator/tree/master/helm
 - Operator Chart
 - Tenant Chart (CR's)
- Zookeeper Operator: https://github.com/pravega/zookeeper-operator/tree/master/charts
 - Zookeeper Operator
 - Zookeeper (CR's)
 
 We also created one for RabbitMQ since only the chart for the operator is provided. We were already thinking of creating one for this Postgres operator but I will take a look at the one from k8s@home. Thanks @Whazor
